packages/slice-machine/lib/models/common/tracker.ts

```diff
@@ -1,38 +1,93 @@
 import Analytics from "analytics-node";
 import { LibraryUI } from "./LibraryUI";
 
-export default class Tracker {
+export interface TrackerOptions {
```

Comment:
Analytics-node doesn't have a type for that ?

Reply:
It does but I don't want to expose all of em

Reply:
So maybe we can use Pick from Typescript ;)

Reply:
nope Pick is used on a specific type, here it's defined this way:

Reply:
```
class Analytics {
  constructor(writeKey: string, opts?: {
    flushAt?: number | undefined,
    flushInterval?: number | undefined,
    host?: string | undefined,
    enable?: boolean | undefined,
    timeout?: number | string | undefined,
  });
```